packages/storybook/stories/table.stories.tsx

```diff
@@ -54,6 +54,62 @@ export const Normal: Story = () => {
     );
 };
 
+interface FooterData {
+    column1: string;
+    column2: string;
+    numbers: number;
+}
+
+export const WithFooter: Story = () => {
+    const columns: TableColumn<FooterData> = [
+        {
+            Header: 'Column 1',
+            accessor: 'column1',
+            Footer: '',
```

Comment:
This is like a backward colSpan. Since the colspan footer is under both the columns 1 and 2, you should move it to the first column since it's where it start.